api/src/test/java/org/openmrs/module/fhir2/providers/r4/MedicationRequestFhirResourceProviderTest.java

```diff
@@ -317,59 +312,6 @@ public void searchMedicationRequest_shouldNotAddRelatedMedicationsForEmptyInclud
 		assertThat(resources.get(0).getIdElement().getIdPart(), equalTo(MEDICATION_REQUEST_UUID));
 	}
 	
-	@Test
-	public void createMedicationRequest_shouldCreateMedicationRequest() {
```

Comment:
Is there a reason we're removing these tests?

Reply:
I removed the create method, which wasn't working, until we have a chance to fix it, see: https://github.com/openmrs/openmrs-module-fhir2/pull/472/files#diff-7c16ac594c8eeabfe06059fce2d86b382b900c02754483dc17c9f5387536170bL72

I guess we could just comment out these tests and pull them back in when we implement, but I don't find these tests to be particularly useful.  @ibacher

Reply:
No that's fine. I didn't realise they weren't working, but now that you mention it, of course that's the case. I much prefer deleting to commenting out.

Reply:
Yeah... At one point I wanted to support MedicationRequests, but then I saw all the stuff Mike went through to implement it in HTMLFormEntry and decided we could wait until there was a real need.